rakefile.rb

```diff
@@ -10,11 +10,13 @@
 xunit_command   = "packages/xunit.runner.console.2.0.0/tools/xunit.console.exe"
 
 solution        = "FakeItEasy.sln"
+dotnet_solution        = "FakeItEasy-NetCore.sln"

```

Comment:
No. I just use a proportional font and don't pay attention to `=` alignment. Will amend. Thanks!